src/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.tsx

```diff
@@ -214,7 +214,7 @@ const AddProfessionalCategoryModal: FC<AddProfessionalCategoryModalProps> = ({
       disableOptions={data.subjects as Array<Partial<SubjectInterface>>}
       handleChange={handleProfessionalSubjectChange(index)}
       handleSubjectDelete={() => handleSubjectDelete(subject._id || '')}
-      key={index}
+      key={subject._id || crypto.randomUUID()}
```

Comment:
Is there any chance for `_id` here to be falsable?

Reply:
Yes, sometimes it's just an emptry string.

Reply:
I think we can remove `crypto.randomUUID()`, since `id=''` will only be used once, when no category has been selected yet.